mentors/elixir/s.json

```diff
@@ -42,5 +42,13 @@
     "link_url": null,
     "avatar_url": "https://avatars0.githubusercontent.com/u/31278?s=460&v=4",
     "bio": "I'm a C++ and Python developer and am in the process of learning Elxir. I always feel that teaching is a great way to learn."
+  },
+  {
+    "github_username": "sleepiejohn",
+    "name": null,
```

Comment:
@sleepiejohn Thank you for submitting your bio and becoming an Exercism mentor. Could you please add your name to your commit because the field cannot be left empty. Thank you.

Reply:
Oh thanks for point it out, I thought it would grab from GitHub!